_docs/administration/oauth-pat-setup.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Set up CSDP as OAuth2 app"
+description: ""
+group: administration
+toc: true
+---
+
+Administrators can create, register and authorize CSDP as a OAuth2 App in GitHub. Users can then authorize access to GitHub instead of generating a personal access token from their Git providers.
+
+To set up OAuth2 authorization for GitHub, as an admin you must create:
+* GitHub OAuth2 Application
+* Secret in runtime cluster with OAuth2 Application credentials
+* Config-map that references the secret
+
+> An OAuth2 Application must be created for every runtime in your deployment.
```

Comment:
Administrator must have config-map and secret in each runtime. But one application can be reused for many runtimes (the same credentials can be used in many secrets)